[2021-04-18 17:00:06,434] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_kc_crime_data-v0.1.check_endpoint_availability 2021-04-18T17:00:02.895656+00:00 [queued]>
[2021-04-18 17:00:06,470] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_kc_crime_data-v0.1.check_endpoint_availability 2021-04-18T17:00:02.895656+00:00 [queued]>
[2021-04-18 17:00:06,470] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-18 17:00:06,471] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-04-18 17:00:06,472] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-18 17:00:06,484] {taskinstance.py:1063} INFO - Executing <Task(HttpSensor): check_endpoint_availability> on 2021-04-18T17:00:02.895656+00:00
[2021-04-18 17:00:06,489] {standard_task_runner.py:52} INFO - Started process 937 to run task
[2021-04-18 17:00:06,494] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_kc_crime_data-v0.1', 'check_endpoint_availability', '2021-04-18T17:00:02.895656+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/fetch_kc_crime_data-v0.1.py', '--cfg-path', '/tmp/tmpxfbhe6jp', '--error-file', '/tmp/tmp0lypfmw2']
[2021-04-18 17:00:06,499] {standard_task_runner.py:77} INFO - Job 11: Subtask check_endpoint_availability
[2021-04-18 17:00:06,592] {logging_mixin.py:104} INFO - Running <TaskInstance: fetch_kc_crime_data-v0.1.check_endpoint_availability 2021-04-18T17:00:02.895656+00:00 [running]> on host 9651c4896f9f
[2021-04-18 17:00:06,680] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=fetch_kc_crime_data-v0.1
AIRFLOW_CTX_TASK_ID=check_endpoint_availability
AIRFLOW_CTX_EXECUTION_DATE=2021-04-18T17:00:02.895656+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-04-18T17:00:02.895656+00:00
[2021-04-18 17:00:06,681] {http.py:102} INFO - Poking: resource/w795-ffu6.json
[2021-04-18 17:00:06,704] {base.py:74} INFO - Using connection to: id: http_data_kcmo_org. Host: https://data.kcmo.org/, Port: None, Schema: , Login: , Password: None, extra: None
[2021-04-18 17:00:06,708] {http.py:140} INFO - Sending 'GET' to url: https://data.kcmo.org/resource/w795-ffu6.json
[2021-04-18 17:00:07,473] {base.py:245} INFO - Success criteria met. Exiting.
[2021-04-18 17:00:07,499] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=fetch_kc_crime_data-v0.1, task_id=check_endpoint_availability, execution_date=20210418T170002, start_date=20210418T170006, end_date=20210418T170007
[2021-04-18 17:00:07,598] {local_task_job.py:146} INFO - Task exited with return code 1
